### DISCUSSION TITLE
yarn v3 zero-install with pm2 cluster mode

### DISCUSSION THREAD

**Discussion by geauser** [yarnpkg/berry#5200] [Q&A]
Hi, I'm using yarn `3.3.1` with zero-installs enabled and I just figured we can't use `yarn` with `pm2` when in [cluster mode](https://pm2.keymetrics.io/docs/usage/cluster-mode/), as it only works with JS files.

Is there anything I can do about it?

Here is what my pm2 `ecosystem.config.js` looks like:

```js
module.exports = {
  apps : [
    {
      name: "api",
      script: "yarn",
      args: ['start'],
      cwd: './modules/api',
      interpreter: "/bin/bash",

      // It works if removes these 2 lines 
      instances: 2,
      exec_mode: 'cluster',
    }
  ]
}
```

**Comment by agongi** [comment]
anyone could support this one?
I did face the same issue
